gcn_classic_to_json/notices/SWIFT_BAT_GRB_POS_ACK/__init__.py

```diff
@@ -1,10 +1,79 @@
+import numpy as np
+
 from ... import utils
 
+flag_descriptions = {
+    0: "A point source was found.",
+    1: "It is a GRB.",
+    2: "It is an interesting src.",
+    3: "It is in the flight catalog.",
+    5: "It is definitely not a GRB.",
+    6: "It is probably not a GRB or Transient(hi bkg level).",
+    7: "It is probably not a GRB or Transient(low image significance; < 7).",
+    8: "It is in the ground catalog.",
+    9: "It is probably not a GRB or Transient(negative bkg slop).",
+    10: "StraTracker not locked so trigger porbably bogus.",
+    11: "It is probably not a GRB or Transient(very low image significance; < 6.5).",
+    12: "It is the catalog of sources to be blocked.",
+    13: "There is a bright star nearby.",
+    14: "This was orginally a SubTresh, but it is now converted to a real BAT_POS.",
+    15: "This is a source that has purposefully been removed from on-board catalog.",
+    16: "This matched a Nearby_Galaxy in the on-board catalog.",
+    28: "There was a temporal coincidence with another event.",
+    29: "There was a spatial coincidence with another event.",
+    30: "This is a test submission",
+}
+
+energy_ranges = [[15, 25], [15, 50], [25, 100], [50, 350]]
+
 
 def parse(bin):
+    bin[15]  # Unused. According to docs: '4 bytes for the future'
+    bin[19]  # Unused. Flags are either internal or equivalent to bin[18]
+    bin[26:36]  # Unused. According to docs: '40 bytes for the future'
+    bin[36]  # Unused. Flags Equivalent to bin[18]
+    bin[38]  # Intentionally omitted. Sun/Moon parameters
+
+    integ_time = bin[14] * 16 / 1000
+
+    lat, lon = bin[16:17].view(">i2")
+
+    soln_status_bits = np.flip(np.unpackbits(bin[18:19].view(dtype="u1")))
+
+    comments = "\n".join(
+        [val for (key, val) in flag_descriptions.items() if (soln_status_bits[key])]
+    )
```

Comment:
Please capture these flags in a machine-readable way, not as comments.

Reply:
I use additional_info to capture the data stored in the `COMMENTS:` fields in the text notices. Some of these are redundant with the data in the notices and can be removed(I've mainly added them for completeness). However, some of the these comments I think may be too complex to easily translate into a field like "StarTracker not locked so trigger probably bogus".